tests/text/modules/heads/test_language_modelling.py

```diff
@@ -2,14 +2,13 @@
 
 import pandas as pd
 import pytest
-import yaml
-from biome.text import TrainerConfiguration, VocabularyConfiguration
+from biome.text import TrainerConfiguration, VocabularyConfiguration, Dataset
 from biome.text import Pipeline
```

Comment:
Maybe do this in one import
```suggestion
from biome.text import Pipeline, TrainerConfiguration, VocabularyConfiguration, Dataset
```

Reply:
Yes, that is much better, I did it in some other files but not in this one for no reason .